src/textual/css/_styles_builder.py

```diff
@@ -332,14 +349,20 @@ def _process_space(self, name: str, tokens: list[Token]) -> None:
                 try:
                     append(int(value))
                 except ValueError:
-                    self.error(name, token, spacing_invalid_value(name, context="css"))
+                    self.error(
+                        name,
+                        token,
+                        spacing_invalid_value_help_text(name, context="css"),
```

Comment:
just a "Black formatting" consequence of adding the missing `_help_text` suffix to these function names